src/tokens/TokenImport.tsx

```diff
@@ -112,10 +207,14 @@ export default function TokenImportScreen(_: Props) {
           <TextInputGroup
             label={t('tokenImport.input.tokenAddress')}
             value={tokenAddress}
-            onChangeText={setTokenAddress}
+            onChangeText={(address) => {
+              setTokenAddress(address)
+              setError(null)
+              setTokenSymbol('')
+              validateContract.reset()
```

Comment:
maybe instead of calling .reset on every character change, we could make the text input disabled if the async request is loading?

Reply:
i think i understand, seems reasonable to leave it like this. are we allowing for the scenario that a user successfully imports a token and we allow them to stay on this screen to import more? i thought that we'd navigate away from the screen on successful import (i see this in https://linear.app/valora/issue/RET-893/import-tokens-screen) ...with that assumption, i was thinking that we might not need the .reset at all

Reply:
> It won't allow rerunning the async function otherwise.

i think that requirement is imposed by https://github.com/valora-inc/wallet/pull/4618/files#diff-c7b8e61e180cc110ced65b99b00a4bb32ccc463744e802ddef49b9d033ed792fR183, i'm not sure what breaks if we remove it. i haven't thought it completely through so if you say we need the .reset then we can keep it